doc/man/bitcoind.1

```diff
@@ -456,6 +456,9 @@ Set maximum size of high\-priority/low\-fee transactions in bytes
 .IP
 Set lowest fee rate (in BTC/kB) for transactions to be included in block
 creation. (default: 0.00001)
+\fB\-bip148\fR
```

Comment:
* 1 orphaned block happens about once a week
* You are suggesting a completely different functionality
* More info: [BIP148](https://github.com/bitcoin/bips/blob/master/bip-0148.mediawiki)